tests/test_unit/test_filtering.py

```diff
@@ -14,7 +14,7 @@
 
 @pytest.fixture(scope="module")
 def sample_dataset():
-    """Return a dict containing a single- and a multi-animal sample dataset"""
+    """Return a single-animal sample dataset"""
```

Comment:
```suggestion
    """Return a single-individual sample dataset"""
```
We use a mix of single- and multi-**animal/individual** throughout `movement`, and I think the use of "animal" in docstrings originated from DLC datasets, but in `movement` we use "individual". What do you think if we retain "animal" whenever it's used in non-`movement` context, e.g. "single-animal DeepLabCut project" and change the rest to "individual", e.g. `invalid_single_animal_csv_file` &rarr; `invalid_single_individual_csv_file` ?

Reply:
Good suggestion, will do.